src/NServiceBus.AcceptanceTests/DelayedDelivery/TimeoutManager/When_timeout_dispatch_fails_on_timeout_data_removal.cs

```diff
@@ -123,11 +122,11 @@ public Task RemoveTimeoutBy(Guid sagaId, ContextBag context)
                 public Task<TimeoutsChunk> GetNextChunk(DateTime startSlice)
                 {
                     var timeouts = timeoutData != null
-                        ? new List<TimeoutsChunk.Timeout>
+                        ? new TimeoutsChunk.Timeout[1]
```

Comment:
As @timbussmann mentioned, the array size is redundant  here, because the array initializer syntax here gets you a length 1 array, but having it here doesn't cause a problem.